tests/registries/legacy/test_legacy_decorators.py

```diff
@@ -174,10 +189,13 @@ def test_on_field_with_all_kwargs(mocker):
     cause = mocker.MagicMock(resource=resource, reason=Reason.UPDATE, diff=diff)
     mocker.patch('kopf.reactor.registries.match', return_value=True)
 
+    when = lambda body: False
```

Comment:
Hm. Does it pass with no `**_`? If yes, that should be worrying, as it should fail here (if called at all).

Reply:
It's technically incorrect and i've fixed it in my next push, but this is just asserting that the decorator propagates `when` to the handler so you could put an object with no __call__ there and it would still pass so i'm not worried that it passes.

Reply:
Ah, I see. I missed that part. Thanks for clarifying.